tutorials/Docking/docking-2020.html

```diff
@@ -54,7 +54,7 @@ <h3>What does this practical cover?</h3>
 </p>
 
 <h2><li><a name="install" id="install">Installing the packages locally</a></li></h2>
-<P>If you are undertaking this practical within the teaching labs in Oxford you can skip this section and head on down to a <a href="#data">Obtaining the data files and checking it all works</a> as everything should already be installed and working.  Ask a Demonstator if not.
+<P>If you are undertaking this practical within the teaching labs in Oxford you can skip this section and head on down to a <a href="#data">Obtaining the data files and checking it all works</a> as everything should already be installed and working.  Ask a Demonstrator if not.
```

Comment:
We might have to clarify this section since they aren't in the lab anymore. If it doesn't apply, let's just drop it (or we can refer back to the readme/setup files).

Reply:
Yep, I can remove it.